File: src/main/java/com/kerosenelabs/billtracker/model/IntroductorySettings.java
package com.kerosenelabs.billtracker.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class IntroductorySettings {
    private String firstName;
    private String lastName;
    private LocalDate birthday;
}

File: src/main/java/com/kerosenelabs/billtracker/model/IntroductorySettings.kt
package com.kerosenelabs.billtracker.model

import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor
import java.time.LocalDate

@Data
@AllArgsConstructor
@NoArgsConstructor
class IntroductorySettings {
    private var firstName: String? = null
    private var lastName: String? = null
    private var birthday: LocalDate? = null
}
